### ISSUE TITLE
Include vertical alignment options in Cover and Columns block

### ISSUE THREAD

**Issue by enriquesanchez** [WordPress/gutenberg#20901]
It'd be great if more blocks such as 'Cover' and 'Columns' had the same vertical alignment features that are found in the 'Media & Text' block:

<img width="365" alt="Screen Shot 2020-03-13 at 16 41 41" src="https://user-images.githubusercontent.com/3276087/76664313-85cc5400-6549-11ea-9f0c-9f8e4c6cbe7f.png">

This feature, if also applied to text child blocks, will allow for more control over text layout and alignment.

See https://github.com/WordPress/gutenberg/pull/20900 for a comparison between what was intended and what was actually possible with Gutenberg.

**Comment by phpbits** [comment]
@enriquesanchez You can use [EditorsKit](https://wordpress.org/plugins/block-options/) plugin to have this vertical alignment option available :) Here's a sample layout using Cover Block + vertical align top.

[https://shareablock.com/blocks/travel-tours-services/](![cover block vertical alignment](https://user-images.githubusercontent.com/3365507/76674428-7d782580-65ea-11ea-8279-01a9856aa0ba.png))

**Comment by phpbits** [comment]
I have this feature added on [EditorsKit](https://wordpress.org/plugins/block-options/) plugin few months ago and here's the preview on how I've implemented the vertical alignment on Cover Block.  I used `verticalAlignment ` attribute similar to what Columns block is using. Thanks!

![Cover Block vertical alignment (24)](https://user-images.githubusercontent.com/3365507/76680777-8ab30580-6626-11ea-987f-56f8b1f9d185.gif)
